atomic_scanners/scanner-rpm-verify/rpm_verify.py

```diff
@@ -196,6 +197,11 @@ def run(self):
         # print error
         if not result:
             result = ["No issues. Libraries and Binaries are intact."]
+            self.json_out["Summary"] = "Libraries and binaries in the image"\
+                " seem intact."
```

Comment:
```s/seem/are```

Reply:
Okay, if you're so sure about this scanner. I don't know detailed working of it so used "seem".

Reply:
We have to be sure about code we are writing and keep on improving per iterations, we dont write in email "you container image build seem to successful."

Reply:
Point noted.